src/components/OneArticle.jsx

```diff
@@ -28,6 +30,59 @@ function OneArticle() {
       });
   }, [article_id]);
 
+  function handleClick(vote, article_id) {
+    const updateVote = { ...singleArticle };
+
+    if (!downvoteClick && !upvoteClick) {
+      updateVote.votes += vote;
+      setSingleArticle(updateVote);
+
+      axios

```

Comment:
It would be better if your axios requests were separated out into functions in their own `api.js` file, and imported where necessary, to keep your code a bit cleaner in your component